### PR TITLE
Adding mutations for the double-odd problem

### PR DESCRIPTION
**Pull request by Sebastian42-5**
For assignment 3 of SF-1

### CODE REVIEW

**Comment by AlemoExe** [comment]
I dont think it can return empty list if the input is invalid (length < 2). You might want to add a clear message or exception for such cases. The rest seems good

**Comment by Sebastian42-5** [comment]
I already put that in the description of the problem

File: list-max-min/io.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "input": [3, 4, 6, 5],
```
[the remark this answers is not in the document]
Comment (Sebastian42-5) [reply list-max-min/io.json]: Thank you. I fixed the problem

**Comment by emaj9** [comment]
But if the elements need to sum up to less than 30... why return more than just the sum of the largest and smallest number? like why return [7, 7, 7, 7] when [7] is also sums to less than 30? @Sebastian42-5

**Comment by Sebastian42-5** [comment]
> But if the elements need to sum up to less than 30... why return more than just the sum of the largest and smallest number? like why return [7, 7, 7, 7] when [7] is also sums to less than 30? @Sebastian42-5

The goal of my problem itself is to make a new list with copies of the sum of the smallest and the largest number of the original list until these numbers get as close to 30 as possible without exceeding 30. Maybe I just wasn't clear with the problem description I could change it if it's necessary

**Review by Monkey-D-Fabio — COMMENTED** [review]
mutation 1 has  an unexpected indent in line 6, 7 and 8 for it to run properly

**Comment by Sebastian42-5** [comment]
> mutation 1 has an unexpected indent in line 6, 7 and 8 for it to run properly. Also third mutation does not produce an error.

I changed it thank you! I also changed mutation 3

**Comment by emaj9** [comment]
Kinda dicey only submitting on the lower end of the mutations, but all the mutations are quite high quality so well done.

**Pull request merged**